tests/php/json-api/test-class.json-api-jetpack-endpoints.php

```diff
@@ -1,38 +1,20 @@
 <?php
+/**
+ * Tests for /sites/%s/categories/slug:%s
+ *
+ * @package Jetpack
+ */
 
 if ( ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) && defined( 'JETPACK__PLUGIN_DIR' ) && JETPACK__PLUGIN_DIR ) {
 	require_once JETPACK__PLUGIN_DIR . 'modules/module-extras.php';
 }
 
 require_jetpack_file( 'class.json-api-endpoints.php' );
 
+/**
+ * Tests for /sites/%s/categories/slug:%s
+ */
```

Comment:
Do we need this comment here? It is the same as in the file header

Reply:
@brbrr I was getting a `Missing doc comment for class` warning. Do you think it's best to remove it?

Reply:
Also I was trying to be consistent with `test-class.json-api-jetpack-site-endpoints.php`

Reply:
Gocha, no worries then :)